packages/ui/src/validators/components/ValidatorsList.tsx

```diff
@@ -134,6 +138,9 @@ const ResponsiveWrap = styled.div`
   @media (min-width: 768px) {
     max-width: calc(100vw - 48px);
   }
+  @media (min-width: 1024px) {
```

Comment:
Small detail but it would be nice to progressively parameterize the media queries:
```suggestion
  @media (min-width: ${BreakPoints.md}px) {
```